code/modules/clothing/under/ties.dm

```diff
@@ -541,13 +541,14 @@
 	hold = /obj/item/storage/internal/tie/white_vest
 
 /obj/item/storage/internal/tie/white_vest
-	storage_slots = 8
+	storage_slots = 1
```

Comment:
this is dumb just load the tools into the vest like it is currently

Reply:
I was told to do it like this reeeeee